### PR TITLE
LPS-170013 Content Selection form is not updated if the "Select This Folder" button is clicked

### PR DESCRIPTION
**Pull request by locpham97**
Hi Team,
This PR is for fixing https://issues.liferay.com/browse/LPS-170013.
Please review and leave your comment.
Thank you,
Loc
cc:@quanhuynhces

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

**Comment by Jeremy-Ch** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java [hunk 1]
```diff
@@ -68,6 +69,34 @@ public KBSelectParentDisplayContext(
 		_initParentData();
 	}
 
+	public String getContentSelectionTitle() throws PortalException {
+		long kbFolderClassNameId = PortalUtil.getClassNameId(
+			KBFolderConstants.getClassName());
+		long contentSelectionId = getParentResourcePrimKey();
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java:75]: Can you rename `contentSelectionId` to `resourcePrimKey`?

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java [hunk 2]
```diff
@@ -68,6 +69,34 @@ public KBSelectParentDisplayContext(
...
+
+		String contentSelectionTitle;
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java:77]: Can you rename `contentSelectionTitle` to simply `title`?

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java:75]: Even better; as this is used only once, can you inline it?

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java (hunk 2 above)
Comment (adolfopa) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java:77]: Alternative, and if possible, can you simply return the title in each of the conditional branches below?
```
if (getResourcePrimKey() == 0) {
    KBFolder kbFolder = ...;

    return kbFolder.getName();
}
else if (...) {
    // similarly with the other branches.
}
...
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java (hunk 2 above)
Comment (quanhuynhces) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java]: I've edited it. Thank you!

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java (hunk 1 above)
Comment (quanhuynhces) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java]: I've edited it. Thank you!

**Comment by quanhuynhces** [comment]
Hi @adolfopa,
I've updated the code following your review.
Thank you!

**Comment by quanhuynhces** [comment]
Hi @adolfopa,
Could you take a look at my new updated code.
Thanks

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp [hunk 3]
```diff
@@ -57,7 +55,7 @@ kbSelectParentDisplayContext.populatePortletBreadcrumbEntries(currentURLObj);
 						).put(
 							"resourcePrimKey", kbSelectParentDisplayContext.getParentResourcePrimKey()
 						).put(
-							"title", parentTitle
+							"title", kbSelectParentDisplayContext.getContentSelectionTitle()
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp:58]: Can you rename `getContentSelectionTitle` to `getSelectedEntryTitle`? Also, rename any variables in the display context to match this.

Sorry I didn't catch this before.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp (hunk 3 above)
Comment (quanhuynhces) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp]: I edited it! 
Thank you

**Comment by quanhuynhces** [comment]
Hey @adolfopa 
I renamed it. Please tell me if you have any concern.
Thank you!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by WYZ-Issac** [comment]
Hi @locpham97, don't forget to fix the conflict.

**Comment by quanhuynhces** [comment]
Hi @WYZ-Issac,
I've fixed the conflict. Could you take a look at this please. 
cc @locpham97

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**